utils/core-modules/contracts/interfaces/IFeatureFlagModule.sol

```diff
@@ -1,23 +1,78 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-/// @title Interface for feature flags
+/**
+ * @title Module for granular enabling and disabling of system features and functions.
+ *
+ * Interface functions that are controlled by a feature flat simply need to add a line to their body:
+ * `FeatureFlag.ensureAccessToFeature(FLAG_ID);`
+ *
+ * If such a line is not present in a function, then it is not controlled by a feature flag.
+ *
+ * If a feature flag is set and then removed forever, consider deleting the line mentioned above from the function's body.
+ */
 interface IFeatureFlagModule {
-    /// @notice Set a feature flag to either allow all or not
+    /**
+     * @notice Emitted when general access has been given or removed for a feature.
+     * @param feature The bytes32 id of the feature.
+     * @param allowAll True if the feature was allowed for everyone and false if it was disallowed for everyone.
```

Comment:
If false it's not really disallowed for everything I think? It's 'only allowed for those in this list'